### ISSUE TITLE
question: not seeing documentation on hover

### ISSUE THREAD

**Issue by ShowMeYourNoobs** [golang/vscode-go#141]
Hi,

I am learning go, using vs code and vscode-go extension (because it is awesome). 

1. I realized that when using gogetdoc, I don't see docs when hovering as shown in the gif below. I can see it only when pressing ctrl+space. 
2. But that doesn't bother me as much as not seeing my own comments. I think that auto-complete shows only go-builtin function comments as it is shown in the gif below. This second thing appears on both gogetdoc and godoc tools. 

If this is not a bug and I am doing something wrong, could you point me to the right direction.

Thanks
![godoc-nok2](https://user-images.githubusercontent.com/52240781/83354271-c7ee8000-a357-11ea-8212-06aefd1ff54c.gif)

**Comment by stamblerre** [comment]
Thanks for the issue report! Can you try running the `gogetdoc` command manually in the terminal? Does it work as expected?

**Comment by ShowMeYourNoobs** [comment]
Hi, I've attached a screenshot of the output.

![Screenshot_3](https://user-images.githubusercontent.com/52240781/83572478-ef954200-a529-11ea-8689-00fd79257161.png)

**Comment by stamblerre** [comment]
The #0 for the position actually refers to the offset in the file, so this is checking the documentation at the `package` statement. Can you try something like `gogetdoc -pos=test.go:#58`?

**Comment by ShowMeYourNoobs** [comment]
WAIT! My bad! 
While I was trying the gogetdoc command as you suggested and at 1 point I got an error saying "no package containing file .\test.go". After that I tried initializing the parent folder of my test.go file with the following command "go mod init <example/folder/structure>". After that, I tried to hover once again and it worked! I guess I have to create a module, for gogetdoc to work? Nevertheless I think this issue is closed and thanks!

![OK](https://user-images.githubusercontent.com/52240781/83574088-06896380-a52d-11ea-8f6a-c62fc423f400.png)
